@ndlib/gatsby-theme-marble/src/components/App/Pages/Portfolio/PortfolioBody/PortfolioItems/EditList/SortableItem/index.js

```diff
@@ -19,6 +20,11 @@ const SortableItem = ({
         cursor: dragged ? 'grabbing' : 'grab',
       }}
     >
+      <img
+        src={dragHandle}
+        alt='drag'
```

Comment:
For accessibility, alt is required, but value should be an empty string since this is more of a decorative element since the whole thing is draggable.
```
alt=''
```

Reply:
since this is a "form" element though if we alt this will we not have an announcement of the sorting ?